CONTRIBUTING.md

```diff
@@ -1,32 +1,23 @@
-# Before you contribute to the SPDX License List
-Please familiarize yourself with the SPDX License List and its supporting documentation, so that you understand the pertinent context around the list itself:
-* [License inclusion principles](DOCS/license-inclusion-principles.md) 
+# Contributing
+The SPDX License List is maintained by the [SPDX legal team](https://spdx.dev/engage/participate/legal-team/). This repo is used for the submission and review of licenses, as well as maintaining [documentation](DOCS) related to the SPDX License List. Suggestions with a larger scope, cross-functional issues, and questions should be directed to the [SPDX-legal mailing list](https://lists.spdx.org/g/Spdx-legal). 
 
-* [Explanation of the fields contained on the list](DOCS/license-fields.md)
+## Submitting a license for consideration
+To submit a license for consideration for the SPDX License List, follow the guidance at [](request-new-license.md). Please be sure to review the relevant documentation prior to you submission, notably, the [license inclusion principles](DOCS/license-inclusion-principles.md) Each license submission requires review and incomplete or inappropriate submissions only take away the precious time of a small number of volunteers from other tasks.
```

Comment:
* "you submission" => "your submission"
* missing period after "license inclusion principles"